### DISCUSSION TITLE
Re-introduce Windows support through WSL2

### DISCUSSION THREAD

**Discussion by sztupy** [CodeYourFuture/curriculum#460] [Ideas]
We stopped supporting Windows in the past because of too many differences between that and UNIX based distributions. However WSL2 support has become much better in recent times, both build speed and convenience is now fairly close to a real UNIX distributions. For trainees who wish to use their own laptops, but cannot install a proper bootable Linux instance, WSL2 is likely a much more convenient alternative than doing a full blown VM install, like using VirtualBox. We might want to explore adding guidance on what WSL2 is, how to install and how it can be used once it's installed

**Comment by SallyMcGrath** [comment]
The thing is, in some regions there's no mentor with a Windows machine and can't really help, and it just wastes masses of class time. It's a huge drag on us. If people want to support a trainee themselves personally with this kind of finicky problem, then fine. But I removed it from the curriculum for this reason.

We have had trainees seriously disadvantaged by using Windows, and miss out on really valuable opportunities. If they insist on their own laptop and they insist on using Windows, then they need to sort it out, so they understand the decision they are making. Otherwise they just choose it because they think it's easier; they don't have the information to understand the choice they are making.
